boards/common/arduino-atmega/Kconfig

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD_COMMON_ARDUINO_ATMEGA
+    bool
+    select HAS_PERIPH_ADC
```

Comment:
How and where will:

```
FEATURES_CONFLICT += periph_rtc:periph_rtt
FEATURES_CONFLICT_MSG += "On ATmega, the RTC and RTT use to the same hardware timer."
```

be handled?

Reply:
I think here the conflict is not of the features but of the modules. So this should be handled when modelling the `periph_rtc` and `periph_rtt` modules. The features are always provided.

Reply:
Ok.